src/Shared/Resources/Strings.shared.resx

```diff
@@ -262,10 +262,26 @@
     <value>Path: {0} exceeds the OS max path limit. The fully qualified file name must be less than {1} characters.</value>
     <comment></comment>
   </data>
+  <data name="JsonParsingError" UESanitized="false" Visibility="Public">
+    <value>MSB5025: Json in solution filter file "{0}" is incorrectly formatted.</value>
+    <comment>{StrBegin="MSB: 5025"}UE: The solution filename is provided separately to loggers.</comment>
+  </data>
+  <data name="MissingSolutionError" UESanitized="false" Visibility="Public">
+    <value>MSB5026: The solution filter file at "{0}" specifies there will be a solution file at "{1}", but that file does not exist.</value>
+    <comment>{StrBegin="MSB: 5026"}UE: The solution filename is provided separately to loggers.</comment>
```

Comment:
Some of these StrBegin are wrong. `{StrBegin="MSB: 5026"}` should be `{StrBegin="MSB5026: "}`

@rainersigwald The purpose of these annotations was to support pseudo localizer tools without breaking the processing of the string by removing the necessary error/warning prefix. I do not know whether pseudo localization is still relevant these days -- if not, they can all be removed -- unlreated to this change of course

Reply:
The loc team does use annotations like these for purposes beyond pseudo loc; I recommend correcting them when you see issues rather than removing them.